ee/vellum_ee/workflows/display/nodes/base_node_display.py

```diff
@@ -386,6 +386,9 @@ def serialize_value(self, display_context: "WorkflowDisplayContext", value: Base
                 "node_id": str(node_class_display.node_id),
             }
 
+        if value.__class__.__name__ == "ParseJsonExpression":
```

Comment:
[nit] why not use `isinstance(value, ParseJsonExpression)` like the rest of the conditions here?